test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1

```diff
@@ -114,5 +114,12 @@ Describe "Stream writer tests" -Tags "CI" {
             $result.Count | Should -Be $returnCount
             (Compare-Object $result $returnValue -SyncWindow 0).length | Should -Be 0
         }
+
+        It "Write-Information accepts `$Null" {
+            $streamPath = Join-Path $testdrive information.txt
+            $null | Write-Information -Tags myTag -ErrorAction Stop -InformationAction SilentlyContinue -InformationVariable i
```

Comment:
Should we also test for directly-passed input?

Reply:
There's existing tests that pass in as parameter.  I don't believe there's a distinct test case here if it's passed in by parameter vs pipeline since the error was at parameter binding and not in the cmdlet